components/application-operator/pkg/kymahelm/application/applicationreleasemanager.go

```diff
@@ -74,20 +75,24 @@ func (r *releaseManager) UpgradeApplicationReleases() error {
 			continue
 		}
 
-		status, description, err := r.upgradeChart(&app)
-		if err != nil {
-			log.Errorf("Failed to upgrade release %s: %s", app.Name, err.Error())
+		r.UpgradeApplicationRelease(&app)
+	}
+
+	return nil
+}
+
+func (r *releaseManager) UpgradeApplicationRelease(app *v1alpha1.Application) {
```

Comment:
I understand that ignoring errors is required here however this is not a good practice to do it outside caller method and unusual with goLang practices. This function may return an error which can be just ignored. I suggest following code:
```
func (r *releaseManager) UpgradeApplicationRelease(app *v1alpha1.Application) error {

	if err != nil {
		log.Errorf("Failed to upgrade release %s: %s", app.Name, err.Error())

			setCurrentStatus(&app, status.String(), description)
		setCurrentStatus(app, status.String(), description)

			err = r.updateApplication(&app)
			if err != nil {
				log.Errorf("Failed to upgrade %s CR: %s", app.Name, err.Error())
			}
		err = r.updateApplication(app)
		if err != nil {
			log.Errorf("Failed to upgrade %s CR: %s", app.Name, err.Error())
		}
	}
 return err
}
```